library/src/helpers/schema.ts

```diff
@@ -101,6 +101,20 @@ export class SchemaHelpers {
     return type;
   }
 
+  static applicatorSchemaName(
+    idx: number,
+    firstCase: string,
+    otherCases: string,
+    title?: string,
+  ) {
+    const suffix = (title !== null && ` ${title}:`) || `:`;
```

Comment:
The title can also be `undefined` it seems. This introduced `Consists of undefined:` for us. So this check needs to be improved.